demo/simple.html

```diff
@@ -584,12 +578,10 @@
 
 }
 
-window.addEventListener('WebComponentsReady', function() {
```

Comment:
Removed this because the ES6 module depending on d2l-table is enough, and no HTMLImports to wait for.